### DISCUSSION TITLE
[Feature] Album addition features

### DISCUSSION THREAD

**Discussion by murtaza-nasir** [immich-app/immich#12490] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to request a plus sign in the web application next to the "appears in" text so that we can directly select albums to put the picture in instead of having to go to the drop-down menu. Similarly, once the album addition dialog box opens, I would like to request the addition of checkboxes, so that multiple albums can be selected. 

Similar easy album selection workflows in the mobile app would also be appreciated. 

![image](https://github.com/user-attachments/assets/3d595bb0-cc80-4a9a-8d6f-1e436de8fc1b)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Only one feature per request please
